pkg/agent/controller/traceflow/traceflow_controller_test.go

```diff
@@ -113,6 +127,15 @@ func newFakeTraceflowController(t *testing.T, initObjects []runtime.Object, netw
 		queue:                 workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "traceflow"),
 		runningTraceflows:     make(map[uint8]*traceflowState),
 	}
+
+	traceflowInformer.Informer().AddEventHandlerWithResyncPeriod(
```

Comment:
which of the new test cases require the event handles to be registered?

Reply:
That doesn't seem right. `TestStartTraceflow` does not require these handlers.
The controller implementation is also asynchronous by nature (handlers may be called asynchronously, handlers add event to workqueue, elements from workqueue are processed asynchronously), and none of your tests are asynchronous, so I would be surprised if these are required.

Reply:
removed, there was different issue.
Please help to review one more time, thanks!